src/com/phronemophobic/membrane/term.clj

```diff
@@ -374,33 +394,42 @@
           (.close (.getInputStream pty))
           (.close (.getOutputStream pty))))))
 
+
 (defn screenshot
-  ([opts]
-   (let [opts (merge default-common-opts
-                     {:line-delay 1e3
-                      :final-delay 10e3
-                      :out "terminal.png"}
-                     opts)
-         {:keys [play width height out line-delay final-delay color-scheme font-family font-size toolkit]} opts
-         term-state (atom {:vt (vt/make-vt width height)})
-         toolkit (if toolkit
-                   toolkit
-                   (load-default-toolkit))
-         font (load-terminal-font toolkit font-family font-size)]
-     (swap! term-state assoc
-            :pty (run-pty-process width height term-state))
-     (doseq [line (string/split-lines (slurp play))]
-       (send-input (:pty @term-state) line)
-       (send-input (:pty @term-state) "\n")
-       (Thread/sleep line-delay))
-
-     (Thread/sleep final-delay)
-     (tk/save-image toolkit
-                    out
-                    (ui/fill-bordered (:background color-scheme) 5
-                                      (term-view color-scheme font (:vt @term-state))))
-     (println (str "Wrote screenshot to " out "."))
-
-     (let [^PtyProcess pty (:pty @term-state)]
-       (.close (.getInputStream pty))
-       (.close (.getOutputStream pty))))))
+  "Take a screenshot after playing a script line by line in a membrane.term terminal.
+  Terminal is not displayed and automatically exits after screenshot is written.
+
+  Requires `opts` map which accepts all options documented in [[run-term]] and:
```

Comment:
We should include all the options here rather than referring to `run-term`. Whenever I use clojure.core.async and the `alts!!` docstring says "like `alts!`", it's really annoying.

Reply:
I'm fine with that.